gstd/src/lib.rs

```diff
@@ -129,7 +129,11 @@
 #![no_std]
 #![warn(missing_docs)]
 #![cfg_attr(
-    all(target_arch = "wasm32", feature = "panic-messages",),
+    all(
+        target_arch = "wasm32",
+        feature = "panic-info-message",
+        feature = "panic-message"
```

Comment:
I guess semantically it could be removed 
```suggestion
```

Reply:
What's the point of enabling nightly feature for panic handler with "no info"?